### PR TITLE
[Wip] Iframe support - FocusToFrame step

### PR DESCRIPTION
**Pull request by amcpanaligan**
# Notes
1. No unit tests yet
2. Tested using the attached scenario file for automaton `hubspot` form

### CODE REVIEW

File: test/scenarios/iframe/iframe.crank.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+steps:
+- step: Navigate to http://www.automatoninc.com/hubspot-test/
+- waitFor: 5
+  step: Click the page element [id=cookie_action_close_header]
+- step: focus on the iframe[id=hs-form-iframe-0] frame
+- step: Fill out input[name=firstname] with Test
+- step: Fill out input[name=lastname] with IFrame
+- step: Fill out input[name=email] with frametest@automatoninc.com
+- step: submit the form by clicking input[type=submit]
+- step: Then the email field on hubspot contact frametest@automatoninc.com should be frametest@automatoninc.com
```
Comment (iamEAP) [inline test/scenarios/iframe/iframe.crank.yml]: - [ ] Nit: recommend adding a `Finally, delete the frametest@automatoninc.com hubspot contact` just to help prevent false positives in the future.

File: src/client/mixins/basic-interaction.ts [hunk 2]
```diff
@@ -205,7 +219,7 @@ export class BasicInteractionAware {
             .catch(e => rej(Error('Submit button still there')));
         }),
         new Promise((res, rej) => {
-          this.client.click(selector)
+          this.client['___currentFrame'].click(selector)
             .then(res)
             .catch((e) => {
               this.client.waitForFunction(
```
Comment (iamEAP) [inline src/client/mixins/basic-interaction.ts]: - [ ] One missing `['___currentFrame']` reference.

Comment (amcpanaligan) [reply src/client/mixins/basic-interaction.ts]: Oh I already fixed that beforehand since my unit tests failed.

Comment (amcpanaligan) [reply src/client/mixins/basic-interaction.ts]: Fixing the `dom.ts` mixin for this comment

**Pull request merged**